core/src/test/java/org/elasticsearch/ingest/IngestClientIT.java

```diff
@@ -219,7 +220,9 @@ public void testPutWithPipelineFactoryError() throws Exception {
         try {
             client().admin().cluster().putPipeline(putPipelineRequest).get();
```

Comment:
not related to the PR, but aren't we missing a fail(); here? if the call succeeds the test succeeds, but we always expect an exception right?